tests/manual-test-examples/p5vr/two-d/example.js

```diff
@@ -26,6 +26,7 @@ function calculate() {
 }
 
 function draw() {
+  setViewerPosition(0, 0, 20);
```

Comment:
Probably better to separate these changes from the openssl documentation changes. This commit can be reverted and then a new branch can be made with a new PR.